README.md

```diff
@@ -169,20 +169,29 @@ enable('<YOUR INSTRUMENTATION KEY GOES HERE>')
 raise Exception('Boom!')
 ```
 
-**Logging requests**
+**Integrating with Flask**
 ```python
 from flask import Flask
-from applicationinsights.requests import WSGIApplication
+from applicationinsights.flask.ext import AppInsights
```

Comment:
Is the WSGI wrapper useful for other frameworks? We might not want to remove it from the docs.

Reply:
Added a link to the middleware in d0b2961. Also updated the middleware's docstring to not use Flask as the code example to avoid potential confusion.